frontend/src/App.tsx

```diff
@@ -13,16 +36,69 @@ function MainPage() {
 }
 
 export default function App() {
+    const classes = useStyles();
     return (
         <Router>
-            <Switch>
-                <Route path='/Tokens'>
-                    <TokenPage />
-                </Route>
-                <Route path='/'>
-                    <MainPage />
-                </Route>
-            </Switch>
+            <Grid
+                container
+                direction='row'
+                justify='space-between'
+                alignItems='center'
+                className={classes.header}
+            >
+                <Typography variant='h5' className={classes.bannerText}>
+                    {'Hi, Jane'}
+                </Typography>
+                {/* Figure out sizes */}
+                <Grid item lg={2} sm={2} xs={2}>
+                    <Grid container direction='row' justify='space-between'>
+                        <Typography className={classes.bannerText}>
+                            {'About'}

```

Comment:
Why is text embedded as javascript string literals?

Reply:
I just prefer it that way

Reply:
I just prefer it that way